public/partials/layer/view.html

```diff
@@ -65,19 +65,25 @@ <h3 class="pull-left" editable-text="_Layer.name">{{ _Layer.name }}</h3>
                             <!--<h5 class="pull-right">Properties</h5>-->
 
                         <layer-object-properties object-type="selectedItem" object="theSelectedElement" on-delete="deleteObject()" on-edit="editObject"
-                        element-types="elementTypes" string-default-aggregation="stringDefaultAggregation" number-default-aggregation="numberDefaultAggregation" 
+                        element-types="elementTypes" string-default-aggregation="stringDefaultAggregation" number-default-aggregation="numberDefaultAggregation"
                         layer="_Layer" on-delete-element="deleteSchemaElement" on-publish-element="elementAdd"></layer-object-properties>
 
                     </section>
 
                     <section id="elementsTab" class="results infinite layer-elements ng-hide " ng-show="tabs.selected == 'elements'">
 
                         <div class="container-fluid">
-                            <button type="button" class="btn btn-primary pull-right"  ng-click="addFolder()" translate>New Folder</button>
-                            <span class="pull-right"> &nbsp; </span>
-                            <button type="button" class="btn btn-primary pull-right"  ng-click="createComposedElement()" translate>New Element</button>
+                            <div id="layerButtonGroup" class="btn-group">
+                                <button id="LayerNewButton" type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" translate>Create
+                                    <span class="caret"></span>
+                                </button>
+                                <ul class="dropdown-menu layerNewButtonDropdown">
+                                    <li><button type="button" class="btn btn-primary layerButton"  ng-click="addFolder()" translate>New Folder</button></li>
+                                    <li><button type="button" class="btn btn-primary layerButton"  ng-click="createComposedElement()" translate>New Element</button></li>
+                                </ul>
+                            </div>
```

Comment:
The appearance of this button dropdown is weird

![Screenshot_2019-05-23 Urungi](https://user-images.githubusercontent.com/306836/58229567-605c2880-7d32-11e9-86e3-3e242de24182.png)

`<button>` should be `<a>` instead. Look at https://getbootstrap.com/docs/3.3/components/#btn-dropdowns-single